recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/wayland/wayland-ivi-extension_1.10.90.bb

```diff
@@ -0,0 +1,26 @@
+# Recipe is cloned from meta-ivi, the Yocto layer for In-Vehicle Infotainment.
```

Comment:
you shouldn't modify the original recipe with this comment. instead add these lines to the commit message

Reply:
Done